entities/jackson/src/main/java/org/ccci/gto/globalreg/entity/jackson/EmailAddress.java

```diff
@@ -16,22 +17,71 @@
     "location",
     "primary",
     "parent_id",
-    "client_integration_id"
+    "client_integration_id",
+    "created_by",
+    "updated_at",
+    "user_entered",
+    "client_updated_at"
 })
 public class EmailAddress {
 
     @JsonProperty("id")
     private String id;
     @JsonProperty("email")
-    private String email;
+    private Email email;
```

Comment:
Correct, this is not backwards compatible. Had their been any dependent clients, I would've mentioned it and this release would've required a new version number.